go.mod

```diff
@@ -0,0 +1,10 @@
+module gsclient-go
+
+go 1.13
+
+require (
+	github.com/google/uuid v1.1.1
+	github.com/gridscale/gsclient-go v2.1.0+incompatible
```

Comment:
I think  `github.com/gridscale/gsclient-go v2.1.0+incompatible` should not be part of the requirements.

Reply:
This is automatically added by gomod as we use it in examples.